programs/bpf/rust/noop/Cargo.toml

```diff
@@ -9,8 +9,10 @@ authors = ["Solana Maintainers <maintainers@solana.com>"]
 repository = "https://github.com/solana-labs/solana"
 license = "Apache-2.0"
 homepage = "https://solana.com/"
+edition = "2018"
 
 [dependencies]
+solana-sdk-rust-utils = { path = "../../../../sdk/bpf/rust-utils", version = "0.15.0" }
```

Comment:
name nit: `solana-sdk-rust-utils` -> `solana-sdk-bpf-utils`?   "rust" is already implicit cause it's a crate.